etna/models/nn/mlp.py

```diff
@@ -0,0 +1,217 @@
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Optional
+from typing import TypedDict
```

Comment:
There is some issues with `TypedDict` - it doesn't exists for python 3.7.

We should import it from typing_extensions